src/components/PromiseDetail/PromiseClarificationLog.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	import { formatThaiDate } from '$lib/date-parser';
+	import type { PromiseClarificationLog } from '$models/promise';
+	import PromiseClarificationAnswer from './PromiseClarificationAnswer.svelte';
+
+	export let partyName: string;
+	export let clarificationLogs: PromiseClarificationLog[] | undefined;
+	$: isClarified = clarificationLogs?.every((log) => !!log.answer);
+</script>
+
+{#if !clarificationLogs || clarificationLogs.length === 0}
+	<div></div>
```

Comment:
We should not render unnecessary empty div. I think we better let the parent (who imports this component) decide if they want to render this component or not. (Because if it is undefined, this component doesn't need to do anything) Then this component doesn't need to handle undefined  `clarificationLogs` cases too!

Reply:
good point na. updated.